docs/hellonode.md

```diff
@@ -221,36 +261,28 @@ First, let’s modify the application. On the development machine, edit server.j
 We can now build and publish a new container image to the registry with an incremented tag:
 
 ```shell
-docker build -t gcr.io/PROJECT_ID/hello-node:v2 . 
-docker push gcr.io/PROJECT_ID/hello-node:v2
+$ docker build -t gcr.io/PROJECT_ID/hello-node:v2 . 
+$ docker push gcr.io/PROJECT_ID/hello-node:v2
 ```
 
 Building and pushing this updated image should be much quicker as we take full advantage of the Docker cache.
 
-We’re now ready for kubernetes to smoothly update our replication controller to the new version of the application:
+We’re now ready for kubernetes to smoothly update our deployment to the new version of the application.  Change the image from gcr.io/PROJECT_ID/hello-node:v1 to gcr.io/PROJECT_ID/hello-node:v2 using `kubectl edit`:
```

Comment:
It's a little cloak-and-dagger what's happening here. How does "edit" know that you're swapping image labels when they aren't mentioned in the example command?